### PR TITLE
Add yaml schedule files for SLE Micro testsuite

### PR DESCRIPTION
**Pull request by chcao**
Add yaml schedule files for SLE Micro testsuite of slem_installation_autoyast and slem_installation_default.

- Related ticket: https://progress.opensuse.org/issues/132992
- Verification run: 
- https://openqa.suse.de/tests/11657764# (autoyast on aarch64)
- https://openqa.suse.de/tests/11657762# (autoyast on s390x)
- https://openqa.suse.de/tests/11657763# (autoyast on x86_64)
- https://openqa.suse.de/tests/11657780# (default on aarch64)
- https://openqa.suse.de/tests/11657781# (default on s390x)
- https://openqa.suse.de/tests/11657782# (default on x86_64)

### CODE REVIEW

**Review by rakoenig — APPROVED** [review]
LGTM.

File: schedule/sle-micro/yam/slem_installation_autoyast.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  Maintainer: QE Yam <qe-yam at suse de>
+  SUSE Linux Enterprise Micro tests
+schedule:
+  - autoyast/prepare_profile
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/logs
+  - console/textinfo
```
Comment (lemon-suse) [inline schedule/sle-micro/yam/slem_installation_autoyast.yaml:12]: I have no idea why need this module.

Comment (chcao) [reply schedule/sle-micro/yam/slem_installation_autoyast.yaml:12]: > I have no idea why need this module.

This module gets useful text-based information from the system and upload it as a log, I think the log is quite useful.

Comment (lemon-suse) [reply schedule/sle-micro/yam/slem_installation_autoyast.yaml:12]: In fact this is not big thing there, while for me it is obvious transactional_update is more important for SLE Micro test. Anyway, we can merge this to make it run on OSD. Later we can enhance this if needed.

**Pull request merged**